internal/indexer/helpers_test.go

```diff
@@ -234,30 +235,9 @@ func findPackageInformationByID(w *capturingWriter, id uint64) (protocol.Package
 	return protocol.PackageInformation{}, false
 }
 
-// findDefintionRangesByDefinitionResultID returns the ranges attached to the definition result with the given
+// findRangesByResultID returns the ranges attached to the definition result with the given
 // identifier.
-func findDefintionRangesByDefinitionResultID(w *capturingWriter, id uint64) (ranges []protocol.Range) {
-	elements := w.elements
-
-	for _, elem := range elements {
-		switch e := elem.(type) {
-		case protocol.Item:
-			if e.OutV == id {
-				for _, inV := range e.InVs {
-					if r, ok := findRangeByID(w, inV); ok {
-						ranges = append(ranges, r)
-					}
-				}
-			}
-		}
-	}
-
-	return ranges
-}
-
-// findReferenceRangesByReferenceResultID returns the ranges attached to the reference result with the given
-// identifier.
-func findReferenceRangesByReferenceResultID(w *capturingWriter, id uint64) (ranges []protocol.Range) {
+func findRangesByResultID(w *capturingWriter, id uint64) (ranges []protocol.Range) {
```

Comment:
Can any other helper functions be simplified w/ this function?

Reply:
I couldn't find a way to reuse it in 5 mins of inspection, but maybe you see one 👀